### ISSUE TITLE
Stats about Penn Libraries to provide readers with context

### ISSUE THREAD

**Issue by dhimmel** [greenelab/library-access#23]
It would be nice in the manuscript to comment on Penn's standing as a research university and Penn Libraries' expenditures.

For example,

+ how much does Penn spend on subscription access to scholarly articles per year?
+ how many articles does Penn subscribe to... what's the average cost per article?
+ what's the average cost per authorized use at Penn?
+ how does Penn's library expenditures compare to other libraries?

I found some info in the [Penn Libraries Facts 2015](http://www.library.upenn.edu/docs/publications/facts3_4_2016.pdf), such as:

> Serials have made the transition to digital and are licensed in bundles that, in the aggregate, consume about 65% of annual information expenditure

@publicus any chance you could find out some of this information... preferably for 2017. I imagine some of it is difficult to calculate, so information that is immediately available is preferred due to time constraints. My main question is how much Penn spends on access to scholarly literature per year. Second, would probably be how articles are accessed via Penn's subscriptions per year.

**Comment by jglev** [comment]
@dhimmel I will ask around about these questions tomorrow!

**Comment by jglev** [comment]
I'm not sure whether data from all of 2017 is available yet in ready form, but I will check on that, as well.

**Comment by dhimmel** [comment]
# ARL Library Investment Index

According to the ARL Library Investment Index, Penn had Total Library Expenditures of $46,905,095 in 2015, which ranks it 15th highest in expenditures among the 144 ARL libraries. This data was downloaded [here](https://www.arlstatistics.org/documents/ARLStats/index16.xls). I've uploaded it to GitHub issues in XLSX format: [`index16.xlsx`](https://github.com/greenelab/library-access/files/1653850/index16.xlsx).

This spreadsheet [has](https://www.arlstatistics.org/analytics) a few variables:

+ Total Library Expenditures (totexp)
+ Salaries & Wages Professional Staff (salprf)
+ Total Library Materials Expenditures (explm)
+ Professional plus Support Staff (totstf)

I think Total Library Materials Expenditures would include subscription journal access. In 2015, Penn expended $18,365,980 in this category. @tamunro does that sound right to you?

**Comment by dhimmel** [comment]
Another stat to help place Penn in the context of research universities is total R&D spending.

[Best Colleges](http://www.bestcolleges.com/features/colleges-with-highest-research-and-development-expenditures/) ranks Penn 13th in total R&D expenditures at $847,077,000. It seems that these numbers are for 2012 and come from the NSF (perhaps https://www.nsf.gov/statistics/srvyrdexpenditures/). Kind of annoying that the NSF turned off its website due to the shutdown.

This [article](https://www.insidehighered.com/news/2016/11/28/new-nsf-data-show-which-universities-are-and-which-down-total-rd-expenditures) has 2015 data.

**Comment by tamunro** [comment]
Yes, this looks good. EXPLM includes serials, but the guide is a bit vague on certain "Electronic Resources". It also includes monographs, which some would argue are not relevant. I'd say they may be - many of them are electronic. The distinction between a "chapter" in a book series and a "journal article" on Springer or Sciencedirect is not substantive, IMO. The ARL stats don't seem to break down print vs electronic monographs.
If @publicus  has the inside scoop on total electronic spending, that would probably be best.

The breakdown is given in "[A Guide to the Machine-Readable Version of the ARL Statistics"](https://www.arlstatistics.org/documents/admin/2012/ddoc2.doc) via [analytics](https://www.arlstatistics.org/analytics#analytics-tabs-0):

> Expenditures for monographs (EXPMONO): ... 
> 
> Expenditures for current serials (EXPSER): ... current serial subscriptions. ...
> 
> Expenditures for other materials (EXPOTH): ... on-site materials other than monographs and serials: ... microforms, audiovisual materials, maps, manuscripts, and similar materials. 
> 
> Expenditures ... for items other than materials (EXPMISC): Some libraries spend part of their materials budget on items such as bibliographic utilities, literature searching, memberships, etc. 
> 
> Total expenditures for materials (EXPLM): EXPLM = EXPMONO + EXPSER + EXPOTH + EXPMISC. 

{i.e. EXPLM = monographs + serials + other + misc}

So yes, it includes serials, but then this is vague:

> Electronic Resources Expenditures
> ... electronic resources and services. These items are not separate from the TOTEXP total;  **most, if not all**, of the expenditures included in TOTEXP and its parts. 
> 
> Expenditures for computer files (EXPCOMPF): ... periodical backfiles, literature collections, one-time costs for JSTOR membership,... 
> 
> Expenditures for electronic serials (EXPESERL): ...
> 
> Expenditures for bibliographic utilities, library (EXPBIBUL)

**Comment by dhimmel** [comment]
# HERD 2016

The NSF website is back up, so we can browse the [results](https://ncsesdata.nsf.gov/herd/2016/) of the Higher Education Research and Development Survey (HERD) for 2016. Specifically, [Table 21](https://ncsesdata.nsf.gov/herd/2016/html/HERD2016_DST_21.html) may be the most pertinent, which is titled "Higher education R&D expenditures, ranked by all R&D expenditures, by source of funds: FY 2016 (Dollars in thousands)." Uploaded here at [`HERD2016_DST_21.xlsx`](https://github.com/greenelab/library-access/files/1657158/HERD2016_DST_21.xlsx). Penn ranked third for all R&D expenditures, which totaled $1,296,429,000.

**Comment by jglev** [comment]
As it turns out, the Penn Libraries' *Selected Facts 2017* publication will come out soon -- if all goes well, around next week in initial PDF format. So these numbers *are* available for 2017.

With that said, as explained below, these numbers should be seen as (accurately) approximate, rather than exact.

> how much does Penn spend on subscription access to scholarly articles per year?

- In 2017, the Libraries spent $20.2 million USD on acquisitions overall.
- Approximately 65% of those funds were used for electronic resources, including journal subscriptions. I was told just now that "electronic resources" also includes ebooks, and that it's difficult to separate journals/articles from ebooks, because many of them come together in bundled packages.

> how many articles does Penn subscribe to... what's the average cost per article?

- I can soon get the total count of electronic journals that we subscribe to; I was told that that's being worked on this week. At that point, we could get approximate average cost per journal. My understanding is that we don't subscribe to *articles* as much as *journal packages,* though, so to get average cost per article, we'd need to get an average number of articles for the journals we subscribe to, or something similar.
  
> what's the average cost per authorized use at Penn?

- In 2017, journal publishers reported to us that 7.3 million download events had occurred, following the [COUNTER Code of Practice](https://www.projectcounter.org/code-of-practice-sections/usage-reports/) that the major publishers have agreed to.
    - I was told that this number has risen 10% over the last 7 years -- i.e., it's quite stable over time.
    - Additionally, the Libraries' [Scholarly Commons](https://repository.upenn.edu/) repository, where copies of many faculty-authored articles are stored for free access, reported 2.1 million download events.
    I don't think that that number is *just* for journal articles, but this still seems a useful number to note, at least here.
    - For ebooks, the metric is chapter views, of which there were 860,000 in 2017. I mention this because part of the acquisitions budget I gave above was used to purchase ebooks.

A back-of-the-napkin calculation: `($20,200,000 * 0.65) / (7,300,000 + 860,000) = $1.609069/download of an article or ebook chapter` (I'm leaving out the Scholarly Commons downloads here, since they're functionally open access, as they're not behind a paywall.)

**Comment by dhimmel** [comment]
@publicus that's fantastic information! Exactly the sort of stats that will provide helpful context. If you remember, comment with the link to "Selected Facts 2017" when it comes out!

**Comment by jglev** [comment]
@dhimmel I'll do so!

**Comment by jglev** [comment]
Just a note to say that, to my knowledge, the *Penn Libraries Facts* booklet is not yet out, but that when it is, I still plant to note the link to it here.
